test/test_eval.py

```diff
@@ -185,4 +193,681 @@ def test_eval_data_split_passage(document_store):
     )
     docs = document_store.get_all_documents(index="haystack_test_eval_document")
     assert len(docs) == 2
-    assert len(docs[1].content) == 56
\ No newline at end of file
+    assert len(docs[1].content) == 56
+
+
+EVAL_QUERIES = ["Who lives in Berlin?", "Who lives in Munich?"]
+EVAL_LABELS = [
+        MultiLabel(labels=[Label(query="Who lives in Berlin?", answer=Answer(answer="Carla", offsets_in_context=[Span(11, 16)]), 
+            document=Document(id='a0747b83aea0b60c4b114b15476dd32d', content_type="text", content='My name is Carla and I live in Berlin'), 
+            is_correct_answer=True, is_correct_document=True, origin="gold-label")]),
+        MultiLabel(labels=[Label(query="Who lives in Munich?", answer=Answer(answer="Carla", offsets_in_context=[Span(11, 16)]), 
+            document=Document(id='something_else', content_type="text", content='My name is Carla and I live in Munich'), 
+            is_correct_answer=True, is_correct_document=True, origin="gold-label")])
+    ]
+
+@pytest.mark.parametrize("retriever_with_docs", ["tfidf"], indirect=True)
+@pytest.mark.parametrize("document_store_with_docs", ["elasticsearch"], indirect=True)
```

Comment:
Any specific reason why you chose elasticsearch here? In context of our discussion about minimizing the memory footprint of tests, it would be an option to go for `memory` here (unless there any doc store functionalities missing)

Reply:
changed all tests that work with InMemoryDocumentStore to `memory`. However there are a few, that do not work. I'm going to investigate this afterwards.